bl-languages/fi_FI.json

```diff
@@ -0,0 +1,242 @@
+	"disabled": "Poissa käytöstä",
+	"cli-mode": "CLI-tila",
+	"command-line-mode": "Komentorivitila",
+	"enable-the-command-line-mode-if-you-add-edit": "Ota käyttöön komentorivitila, jos lisää, muokkaat tai poistat sivuja tai blogikirjoituksia suoraan tiedostojärjestemästä",
+
+	"configure": "Asetukset",
+	"uninstall": "Poista",
+	"change-password": "Muuta salasana",
+	"to-schedule-the-post-just-select-the-date-and-time": "Ajastaaksesi lähetyksen, valitse päivä ja aika.",
+	"write-the-tags-separated-by-commas": "Kirjoita tagit erotetuna pilkuilla.",
+	"status": "Tila",
+	"published": "Julkaistu",
+	"scheduled-posts": "Ajastetut blogikirjoitukset",
+	"statistics": "Tilastot",
+	"name": "Nimi",
+	"email-account-settings":"Sähköpostin asetukset",
+	"sender-email": "Lähettäjän sähköpostiosoite",
+	"emails-will-be-sent-from-this-address":"Sähköpostit tullaan lähettämään tästä osoitteesta.",
+	"bludit-login-access-code": "BLUDIT - Kirjautumisen tunnistautumiskoodi",
+	"check-your-inbox-for-your-login-access-code":"Tunnistautumiskoodi on lähetetty sinulle sähköpostilla",
+	"there-was-a-problem-sending-the-email":"Sähköpostin lähettämisessä tapahtui ongelma",
+	"back-to-login-form": "Takaisin",
+	"send-me-a-login-access-code": "Lähetä minulle tunnistautumiskoodi",
+	"get-login-access-code": "Lähetä tunnistautumiskoodi",
+	"email-notification-login-access-code": "<p>Tämä on ilmoitus sivustoltasi {{WEBSITE_NAME}}</p><p>Jos pyysit kirjautumisen tunnistautumiskoodia, paina tästä linkistä:</p><p>{{LINK}}</p>",
+	"there-are-no-scheduled-posts": "Ei ajastettuja blogikirjoituksia",
+	"show-password": "Näytä salasana",
+	"edit-or-remove-your=pages": "Lisää, muokkaa ja poista sivujasi",
+	"edit-or-remove-your-blogs-posts": "Lisää, muokkaa ja poista blogikirjoituksiasi",
+	"general-settings": "Yleiset asetukset",
+	"advanced-settings": "Lisäasetukset",
+	"manage-users": "Hallitse käyttäjiä",
+	"view-and-edit-your-profile": "Näytä ja muokkaa oma profiilisi",
+
+	"password-must-be-at-least-6-characters-long": "Salasanan täytyy olla vähintään 6 merkkiä pitkä",
+	"images": "Kuvat",
+	"upload-image": "Lähetä image",
+	"drag-and-drop-or-click-here": "Klikkaa tästä, tai vedä kuva tänne",
+	"insert-image": "Lisää kuva",
+	"supported-image-file-types": "Tuetut kuvatyypit",
+	"date-format": "Päivämäärän muoto",
+	"time-format": "Ajan muoto",
+	"chat-with-developers-and-users-on-gitter":"Keskustele muiden kehittäjien ja käyttäjien kanssa [Gitterissä](https://gitter.im/dignajar/bludit)",
+	"this-is-a-brief-description-of-yourself-our-your-site":"Tämä on lyhyt kuvaus sinusta, tai sivustostasi. Voit muuttaa tätä hallintapaneelista kohdasta Asetukset->Lisäosat, ja muokkaa lisäosan \"About\" asetuksia",
+	"profile-picture": "Profiilikuva",
+	"the-about-page-is-very-important": "Tietoja-sivu on tärkeä työkalu mahdollisille asiakkaillesi. Niille asiakkaille, jotka miettivät, kuka on tämän mahtavan sivuston takana, sinun Tietoja-sivu on ensimmäinen paikka josta katsoa.",
```

Comment:
"the-about-page-is-very-important": "Tietoa-sivu on tärkeä ja tehokas työkalu mahdollisille asiakkaille ja yhteistyökumppaneille. Heille, jotka miettivät, kuka tai ketkä ovat tämän sivuston takana, Tietoa-sivu on ensisijainen tiedonlähde."